pom.xml

```diff
@@ -200,7 +200,7 @@ flexible messaging model and an intuitive client API.</description>
     <protobuf-maven-plugin.version>0.6.1</protobuf-maven-plugin.version>
     <exec-maven-plugin.version>1.6.0</exec-maven-plugin.version>
     <aspectj-maven-plugin.version>1.11.1</aspectj-maven-plugin.version>
-    <license-maven-plugin.version>3.0</license-maven-plugin.version>
+    <license-maven-plugin.version>4.0.rc2</license-maven-plugin.version>
```

Comment:
I wonder to know should we use a stable version for that? `rc` seems it is not a stable version.

Reply:
3.0 was released in August 2016, so that's one of the reasons to get on 4.0.rc2 which was release in August 2020. I wouldn't be concerned that it's a rc version. The commit history of license-maven-plugin can be viewed at https://github.com/mycila/license-maven-plugin . It seems that there are quite a few fixes included in 4.0.rc2 compared to 3.0

Reply:
Since this is a dev tool and not a runtime dependency, I don't see any issue to upgrade to an RC version